osu.Game/Screens/Play/SongProgress.cs

```diff
@@ -130,10 +139,15 @@ protected override void Update()
             if (objects == null)
                 return;
 
-            double progress = ((AudioClock?.CurrentTime ?? Time.Current) - firstHitTime) / lastHitTime;
+            double currentTime = AudioClock?.CurrentTime ?? Time.Current;
+            double progress = (currentTime - firstHitTime) / lastHitTime;
 
             bar.UpdatePosition((float)progress);
             graph.Progress = (int)(graph.ColumnCount * progress);
+
+            info.TimeCurrent = TimeSpan.FromMilliseconds(currentTime).ToString(@"m\:ss");

```

Comment:
yep, raw values should be passed in and formatting should be handled by the class itself.